src/js/media-time-display.js

```diff
@@ -76,6 +76,7 @@ class MediaTimeDisplay extends MediaTextDisplay {
 
   disconnectedCallback() {
     this.disable();
+    super.disconnectedCallback();
```

Comment:
fixed an existing bug here with the help of a test, un-associating a state receiver